tasks/noauto_create_timer.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Create borgbackup timer
+  when:
+    - install_backup is not defined or install_backup
+    - borgmatic_timer is defined and borgmatic_timer == "systemd"
+  tags:
+    - install_backup
+  block:
+    - name: Add systemd.timer for borgmatic (large repo, create and check separate)
+      when: borgmatic_large_repo
+      block:
+        - name: Copy systemd files
+          ansible.builtin.template:
+            src: "{{ item.src }}"
+            dest: "{{ item.dest }}"
+            owner: root
+            group: root
+            backup: true
+            mode: "{{ item.mode }}"
+          with_items:
+            - { src: "backup_large_repo_check.timer.j2", dest: "/usr/lib/systemd/system/backup_large_repo_check.timer", mode: "0644" }
```

Comment:
Good thinking to install those non-editable files to `/usr/lib/systemd`.

How about adding a short note in the README that settings can be [overridden](https://askubuntu.com/questions/659267/how-do-i-override-or-configure-systemd-services) in `/etc/systemd`?